elements/pfe-accordion/src/pfe-accordion-panel.scss

```
@@ -27,7 +27,7 @@
   :host([pfe-disclosure="true"]) & {
     @include pfe-print-local((
       panel-container: (
-        Padding: 0 calc(#{pfe-local(Padding, $region: base)} * 3) pfe-local(Padding, $region: base) calc((#{pfe-local(Padding, $region: base)} * 1.5) - 4px)
+        Padding: 0 calc(#{pfe-local(Padding, $region: base)} * 1.5) pfe-local(Padding, $region: base) calc((#{pfe-local(Padding, $region: base)} * 1.5))
```

```suggestion
        Padding: 0 calc(#{pfe-local(Padding, $region: base)} * 1.5) pfe-local(Padding, $region: base)
```

@deborapozzebon This is looking good!  You can use the shorthand here to make it a little less verbose if you like.  Will result in the same output.